umm-spec-lib/test/cmr/umm_spec/test/umm_g/granule.clj

```diff
@@ -9,6 +9,7 @@
    [cmr.common.test.test-check-ext :refer [defspec]]
    [cmr.umm-spec.umm-spec-core :as core]
    [cmr.umm.test.generators.granule :as gran-gen]
+   [cmr.umm.umm-collection :as umm-c]
```

Comment:
It might be worth moving some of this common functionality into a utility namespace in the future. Using collection-specific code in granule-specific namespaces is bound to get confusing and will probably lead to duplication

Reply:
This is to reference the RangeDateTime type that is defined in `umm-collection`.